config.dev.toml

```diff
@@ -154,7 +147,7 @@ StatsExporter = "prometheus"
     [Storage.Disk]
         # RootPath is the Athens Disk Root folder
         # Env override: ATHENS_DISK_STORAGE_ROOT
-        RootPath = "/path/on/disk"
+        RootPath = "/Users/208581/Desktop/ans"
```

Comment:
Let us keep it empty rather than a folder that does not exist.

Reply:
This was a slip up while testing things, will put it back to what it was can be changed in a future pr. trying to keep it as small as possible haha.